### DISCUSSION TITLE
Docker API, containers/{id}/archive returns only tar archives?

### DISCUSSION THREAD

**Discussion by RamiresOliv** [moby/moby#46820] [Q&A]
Hello, I would like to know if there is any other way to export the file other than as a tar file. And somehow get this information in UTF8, in some way that I can put it in a file on my computer other than a .tar .. (basically unzip the file) but I have no idea how to do that.

like this:
![imagem_2023-11-14_224905625](https://github.com/moby/moby/assets/69019082/8cb97384-6ac1-4c4d-b7da-4a0e2fae2a50)

**Comment by neersighted** [comment]
The interface is defined in terms of tar archives; I would suggest that if that is cumbersome, you consider using a programming language that lets you work with tar files in memory. For example in Go, you can use https://github.com/cpuguy83/tar2go to trivially access files as you would with `fs.FS` from an in-memory tar.
